### PR TITLE
Show the Ctrl-Alt-Delete action

### PR DESCRIPTION
**Pull request by lonicerae**
- Since the man page of 'systemctl' says it can show properties of th manager.

### CODE REVIEW

**Comment by lonicerae** [comment]
Hello,

Just backported it on RHEL8, the output is as follows:

~~~
# uname -a
Linux rhel8u3.example.com 4.18.0-240.el8.x86_64 #1 SMP Wed Sep 23 05:13:10 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux

# systemctl show | grep -i ctrl
CtrlAltDelBurstAction=2
~~~

Please help to review it. Thank you.

Best regards,
Flos

File: src/core/dbus-manager.c [hunk 1]
```diff
@@ -2604,6 +2604,7 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_PROPERTY("DefaultTasksMax", "t", bus_property_get_tasks_max, offsetof(Manager, default_tasks_max), 0),
         SD_BUS_PROPERTY("TimerSlackNSec", "t", property_get_timer_slack_nsec, 0, SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("DefaultOOMPolicy", "s", bus_property_get_oom_policy, offsetof(Manager, default_oom_policy), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("CtrlAltDelBurstAction", "x", bus_property_get_long, offsetof(Manager, cad_burst_action), SD_BUS_VTABLE_PROPERTY_CONST),
```
Comment (poettering) [inline src/core/dbus-manager.c]: please return a string here. the property_get_emergency_action() helper makes that easy.

**Comment by lonicerae** [comment]
Thanks @poettering ! Just  tried to update the code a little, please help to review it again. Also tested on my RHEL8.3 as follows:

~~~
[root@rhel8u3 ~]# cat /etc/systemd/system.conf |grep Ctrl
#CtrlAltDelBurstAction=reboot-force
CtrlAltDelBurstAction=poweroff-force
[
root@rhel8u3 ~]# systemctl show --all | grep Ctrl
CtrlAltDelBurstAction=poweroff-force
~~~

File: man/org.freedesktop.systemd1.xml [hunk 2]
```diff
@@ -492,6 +492,8 @@ node /org/freedesktop/systemd1 {
       readonly t TimerSlackNSec = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly s DefaultOOMPolicy = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly x CtrlAltDelBurstAction = ...;
```
Comment (poettering) [inline man/org.freedesktop.systemd1.xml]: type must be "s".

maybe regenerate this whole file

**Comment by lonicerae** [comment]
Thanks for the tips!Already regenerated the docs. Please help to check them again.

**Pull request merged**